_posts/2022-11-21-trino-summit-2022-recap.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: "Trino Summit 2022 Recap"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/stage.jpg
+---
+
+Trino Summit 2022 was in a word, invigorating. I’m still coming off the high 
+from the amount of energy I gained from being at this summit, meeting many of
+you face-to-face for the first time. Most surprisingly, I learned that Trino
+contributor James Petty from AWS was actually not famous painter
+[Bob Ross](https://en.wikipedia.org/wiki/Bob_Ross).
+
+<!--more-->
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/blog/trino-summit-2022/james-petty.png"/><br/>
+</p>
+
+If you’ve ever planned a conference, you know that there are a lot of details to
+iron out, and you can be left exhausted by the end. After this year’s Trino
+Summit though, rather than being worn out, I felt like it ended too quickly and
+I simply wanted more time to chat with everyone. A single day was simply not
+enough, and now all I can think about is the next summit. We not only got to
+hear an incredible lineup of talks and discussions from first-time Trino Summit
+speakers like Apple, Shopify, and Lyft, but also had many engaging discussions
+outside the auditorium.
+
+<img width="75%" src="/assets/blog/trino-summit-2022/swag.jpg"/>
+<img width="75%" src="/assets/blog/trino-summit-2022/authors.jpg"/>
+<img width="75%" src="/assets/blog/trino-summit-2022/talking-1.jpg"/>
+<img width="75%" src="/assets/blog/trino-summit-2022/talking-2.jpg"/>
+
+There were cross-community discussions between Delta Lake, Airflow, and Alluxio
+about how to turbo-charge Trino integrations with these communities. There were
+many companies talking about best practices and gotchas while migrating from
+Hive to Iceberg or Delta Lake. Others wanted to learn how to use fault-tolerant
+execution. I spoke with managers of companies like LinkedIn and Bloomberg who
+wanted to help develop their engineers to get more involved with contributing to
+Trino. We all finally got to see the faces of people we had been talking to for
+the past two to three years for the first time. People were getting their free
+copies of Trino: The Definitive Guide signed by Manfred, Matt, and Martin and
+brought home other swag. After a long day of talks, we wrapped Trino Summit up
+with two happy hours on the roof of the Commonwealth club watching the sunset
+over the San Francisco bay bridge.
+
+<img width="75%" src="/assets/blog/trino-summit-2022/speech.jpg"/>
+<img width="75%" src="/assets/blog/trino-summit-2022/happy-hour.jpg"/>
+
+## Session summaries
+
+I would like to quickly summarize a few short takeaways I had from each talk at
+the summit. I highly recommend you watch the full videos on the Trino YouTube
+which are linked in the titles:
+
+* [<i class="fab fa-youtube" style="color: red;"/> Keynote: State of Trino](https://www.youtube.com/watch?v=mUq_h3oArp4)
```

Comment:
Remove the bullet item here and for all other titles for the sessions below .. it already breaks up everything nicely now without that